src/NUnitTestAdapter/Properties/AssemblyInfo.cs

```diff
@@ -21,5 +21,5 @@
 [assembly: ComVisible(false)]
 
 [assembly: Guid("c0aad5e4-b486-49bc-b3e8-31e01be6fefe")]
-[assembly: AssemblyVersion("1.2.0.1")]
-[assembly: AssemblyFileVersion("1.2.0.1")]
+[assembly: AssemblyVersion("1.9.9.0")]
```

Comment:
Yes it should. 
I just kept it at 1.9.9 while beta'ing it, as vsix'es dont have any beta concept like nuget packages has.